dashboard/pages/primary_screening/callbacks.py

```diff
@@ -51,12 +51,40 @@ def upload_bmg_data(contents, names, last_modified, stored_uuid, file_storage):
 
     return (
         html.Div(
-            [
-                html.H5("Loaded files"),
+            className="row",
+            children=[
+                html.H5(className="text-center", children="Loaded files"),
                 html.Hr(),
-                html.Ul(children=[html.Li(i) for i in names]),
-                html.Hr(),
-            ]
+                html.Div(
+                    className="row",
+                    children=[
+                        html.Div(
```

Comment:
To simplify this you can use

```python
NUM_COLS = 4
...
children = [
    html.Div(
        className="col",
        children=html.Ul(
            children=[html.Li(name) for name in names[i::NUM_COLS]]
        )
    )
    for i in range(NUM_COLS)
]
```

avoids code duplication and allows to easily customize number of columns

Reply:
Even better - considerind you use it for the other input as well, you probably should just make it a function that intakes number of columns and the list of names, and returns generated html elements